src/nm.c

```diff
@@ -700,6 +700,9 @@ write_nm_conf_access_point(NetplanNetDefinition* def, const char* rootdir, const
         if (ap->bssid) {
             g_string_append_printf(s, "bssid=%s\n", ap->bssid);
         }
+        if (ap->hidden) {
+            g_string_append(s, "hidden=TRUE\n");
```

Comment:
Just a little stylistic nitpick: Could you please change this to "hidden=true", we use lower case boolean values in the rest of the code, so I'd like to keep this consistent.

Reply:
No problem - quick fix!